test/sass/mixins--spacing.spec.scss

```diff
@@ -0,0 +1,427 @@
+@import "../../node_modules/sass-true/sass/true";
+@import "../../source/css/sass/_mixins--spacing.scss";
+
+@include describe('_padding-left [Mixin]') {
```

Comment:
If https://github.com/libero/patterns-core/pull/8#discussion_r217793364 is true, these shouldn't be tested directly as `padding` is the unit.

Reply:
Agreed. Will remove tests for `_padding-left` and `_padding-right`.